### ISSUE TITLE
ReferenceError: webpackJsonp is not defined

### ISSUE THREAD

**Issue by ErichBSchulz** [kriasoft/react-starter-kit#1406]
Hi,

I am hitting:

```
GET http://138.68.238.144:3002/assets/vendor.js net::ERR_CONTENT_LENGTH_MISMATCH
about.chunk.js:1 Uncaught ReferenceError: webpackJsonp is not defined(anonymous function) @ about.chunk.js:1
client.js:1 Uncaught ReferenceError: webpackJsonp is not defined
```

on a vanilla install (under both node 8.4 and 8.5) on a vanilla install on a docker container running in a "digital ocean droplet". It works fine on a docker container running on my local machine and I have no clue how to start unpicking this.

you can see it [here](http://138.68.238.144:3002) currently.

any clues?

**Comment by valeeum** [comment]
That usually means the static, compiled js file created by build process  doesn’t exist on server.  Are you by chance running this behind a load balancer? If so, it’s possible that each instance is generating a unique chunk ID which is resulting in the 404 error.

**Comment by ErichBSchulz** [comment]
Thanks for the thought but no load balancing. This is a single droplet.

**Comment by tim-soft** [comment]
Seems like an issue with node 8.x
https://github.com/nodejs/node/issues/15082

Try the latest 7.x and see if the issue goes away

**Comment by langpavel** [comment]
I have different thought: You are running development build instead of production. Why the port is 8002? This is for BrowserSync in development.

If you configured port explicitly by `NODE_PORT` it's not point to this..

Did you deploy correctly?

**Comment by ErichBSchulz** [comment]
hi all thanks for these thoughts - sorry I've put this on the back burner - hopefully I will get back to it in a month or two - happy for you to close this issue if none others have reported issues.

**Comment by langpavel** [comment]
Ok, I will close, if anybody have trouble like this, feel free to reopen :-) Happy coding :heart:

**Comment by ted-lee** [comment]
I have encountered the same problem whenever I use "ngrok" to expose my local server (launched in development mode) to the public internet for testing. I have fixed it by just upgrading node from v8.4 to v8.9.4.
